frontend/src/views/Transactions.vue

```diff
@@ -3,75 +3,115 @@
-              {{ transaction.get('name') }} <small class="font-normal text-xs">({{ transaction.symbols.length }} stocks)</small>
-            </div>
-
-            <div class="text-gray-500  text-sm mb-2">
-              {{ $t('transaction.updated') }} {{ dayjs(transaction.get('updatedAt')).fromNow() }}
-            </div>
-
-
-            <div
-              :class="transaction.percentChange >= 0 ? 'bg-green-100 text-green-800' : 'bg-red-100 text-red-800'"
-              class="min-w-min p-3 text-xl rounded-lg font-bold"
-            >
-              <span v-if="transaction.percentChange >= 0">+</span>{{ transaction.percentChange.toFixed(2) }} %
-            </div>
-          </div>
+        <AppLink
+          :ticker="slotProps.row['Ticker']"
+          to="ticker_details"
+        >
+          {{ slotProps.row['Position'] }}
+        </AppLink>
+      </template>
+      <template
+        #Type="slotProps"
+      >
+        <div class="flex gap-2">
+          <svg
+            class="h-6 w-6"
+            :class="slotProps.row['Type'] === 'BUY' ? 'transform rotate-180 text-yellow-500' : 'text-blue-500'"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M11 15l-3-3m0 0l3-3m-3 3h8M3 12a9 9 0 1118 0 9 9 0 01-18 0z"
+            />
+          </svg>
+          {{ slotProps.row['Type'] }}
         </div>
-      </div>
-    </div>
+      </template>
+    </DataTable>
   </div>
 </template>
 
 <script lang="ts">
-import {defineComponent, onMounted, onUnmounted, ref} from 'vue'
+import {computed, defineComponent, onMounted, onUnmounted, reactive} from 'vue'
+import {Transaction} from '../models'
 import dayjs from 'dayjs'
-import {Watchlist} from '../../../common/models/Watchlist'
-
+import DataTable, {TableCellType, TableConfig} from '../components/DataTable.vue'
+import AppLink from '../components/router/AppLink.vue'
 
 export default defineComponent({
+  components: {AppLink, DataTable},
   setup () {
-    const watchlists: Watchlist[] = ref([])
-    const newWatchlistName        = ref('')
-    let liveSubscription          = null
-
-    const createList = async () => {
-
-      const watchlist = new Watchlist()
-
-      watchlist.set('name', newWatchlistName.value)
-      await watchlist.save()
-      newWatchlistName.value = ''
-    }
-
-    const show = async (watchlist: Watchlist) => {
-
-      watchlist.percentChange = (Math.random() * 10) - 3.5
-      watchlist.symbols       = await watchlist.relation('symbols').query().find()
-    }
-
+    const transactions: Transaction[] = reactive([])
+    let liveSubscription              = null
 
     onMounted(async () => {
-      const q = new Parse.Query(Watchlist)
+      const q          = new Parse.Query(Transaction)
+      q.descending('createdAt')
```

Comment:
`createdAt` should not be used here. It refer to the document creation.

For example if you add a transaction that occured a month ago `createdAt = now`, and `date = 6 month ago`

(I'll rename the field `date` for `transactionDate` or similar)